pkg/jsonlog/jsonlog_marshalling.go

```diff
@@ -101,6 +101,15 @@ func (mj *JSONLog) MarshalJSONBuf(buf *bytes.Buffer) error {
 		buf.WriteString(`"stream":`)
 		ffjsonWriteJSONString(buf, mj.Stream)
 	}
+	if len(mj.Tag) > 0 {
```

Comment:
Please add a case that uses `tag` to the list of testcases for this function

Reply:
Done. Added an extra log line with a tag to the existing unit test for it.